pkg/containerd/importer/importer.go

```diff
@@ -0,0 +1,294 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package importer
+
```

Comment:
//This code reuses the docker import code from containerd/containerd#1602. 
// It has been simplified a bit and garbage collection support was added. 
// If a library/helper is added to containerd in the future, we should switch to it.

Reply:
Will do.

Reply:
Done.